Sources/CocoaLumberjack/DDLog.m

```diff
@@ -1242,9 +1242,7 @@ - (void)setLogFormatter:(id <DDLogFormatter>)logFormatter {
         }
     };
 
-    dispatch_queue_t globalLoggingQueue = [DDLog loggingQueue];
-
-    dispatch_async(globalLoggingQueue, ^{
+    dispatch_async([DDLog loggingQueue], ^{
```

Comment:
Nowadays `DDLog.loggingQueue` is more "idiomatic" in terms of interoperability/readability, no? 
"Shape of Swift", hah.

Reply:
Done.